config/packages/flysystem.yaml

```diff
@@ -0,0 +1,7 @@
+# Read the documentation at https://github.com/thephpleague/flysystem-bundle/blob/master/docs/1-getting-started.md
+flysystem:
+    storages:
+        default.storage:
+            adapter: 'local'
+            options:
+                directory: '%kernel.project_dir%/var/storage/default'
```

Comment:
I would keep here the recipe default as it better to be as near as possible to symfony defaults.

Projects upgrading need to change the directory like documented in: https://github.com/sulu/sulu/blob/3.0/UPGRADE.md#updating-to-flysystem-3